src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -199,6 +199,11 @@ let PostThreadItemLoaded = ({
     return makeProfileLink(post.author, 'post', urip.rkey, 'reposted-by')
   }, [post.uri, post.author])
   const repostsTitle = _(msg`Reposts of this post`)
+  const quotesHref = React.useMemo(() => {
+    const urip = new AtUri(post.uri)
+    return makeProfileLink(post.author, 'post', urip.rkey, 'quotes')
+  }, [post.uri, post.author])
+  const quotesTitle = _(msg`Quotes of this post`)
```

Comment:
Weird that we did this too lol but also minor unless you feel like moving these down directly to the props.

Reply:
ehh I feel like we should keep it consistent

Reply:
Oh, yea unless you redo the whole thing don't worry about it. But I'm just wondering why we did it this way in the first place. (Definitely fine to leave it alone though)